pyccel/parser/syntax/openmp.py

```diff
@@ -413,7 +1415,32 @@ def expr(self):
                OmpPrivate,
                OmpReduction,
                OmpSchedule,
-               OmpShared]
+               OmpShared,
+               OmpCriticalName,
+               OmpFilter,
+               OmpinReduction,
+               OmpNumTasks,
+               OmpGrainSize,
+               OmpUntied,
+               OmpMergeable,
+               OmpNogroup,
+               OmpPriority,
+               OmpAtomicClause,
+               AtomicMemoryClause,
+               OmpDepend,
+               FlushList,
+               OmpFinal,
+               OmpCancelType,
+               OmpMap,
+               OmpNumTeams,
+               OmpThreadLimit,
+               OmpForSimd,
+               OmpMaskedTaskloop,
+               OmpPSections,
+               OmpTaskloopSimd,
+               OmpDistributeCombined,
+               OmpTargetParallel,
+               OmpTargetTeams]
 
 omp_classes = [Openmp, OpenmpStmt] + omp_directives + omp_clauses
```

Comment:
Do you know if there's a reason why these last 3 are lists? I agree with you that they should be tuples. They should be immutable

Reply:
i don't think there is a reason, I changed them to a tuple and everything is working normally.